Changes

```diff
@@ -1,6 +1,8 @@
 Revision history for {{$dist->name}}
 
 {{$NEXT}}
+  - Test::Alien::Diag should not handle undef fields more
```

Comment:
```s/not//;```

Reply:
thanks for catching that I meant to type "now" 5f47dba